contribute.md

```diff
@@ -9,3 +9,33 @@ Join our [DAO Discord](https://discord.gg/42xFV68uEf) and make sure you fill up
 *Note: The onboarding process might be subject to change in the future. Worry not, just ask around in the discord and our helpful community will guide you through the onboarding process.*
 
 ## Contributor Workflow
+
+Every contributor should follow this workflow when updating the documentation. At a high level, the workflow is as follows:
+
+1. Clone the GitHub repository onto your local machine.
```

Comment:
This assumes you already have contributor access. Should we add information around the process of onboarding? Like we need your github username and you get added to the team and then you are good to go.

Reply:
Ok, I made a commit to include this. I left out some details here, just direct them to our DAO discord server to keep the instruction brief.